README.rst

```diff
@@ -49,8 +49,6 @@ Grid fitters
 ------------
 
 - ``GridFit``: Abstract base class for grid fitters
-- ``Delaunay2DGridFit``: Interpolate scalar function defined on 2-D grid,
-  based on Delaunay triangulation
```

Comment:
Thinking about it again, I'll add a changelog section in the long description...